packages/klighd-core/src/proxy-view/proxy-view-cluster.tsx

```diff
@@ -0,0 +1,150 @@
+import { VNode } from "snabbdom";
+
+/** Returns the rendering of clusters. */
+export function getClusterRendering(id: string, numProxies: number, size: number, x: number, y: number): VNode {
+    const squareSize = size * 0.9;
+    const squareOffset = size * 0.1;
+    const textSize = size * 0.5;
+    return JSON.parse(
+        `{
+            "sel": "g",
+            "data": {
+                "ns": "http://www.w3.org/2000/svg",
+                "attrs": {
+                    "id": "keith-diagram_sprotty_$${id}",
```

Comment:
Does this guarantee unique IDs? Otherwise some $-separated proxy view prefix would be good

Reply:
I'm pretty sure it should be unique already since there's no $root and when it's called by the clustering the given id definitely makes it unique. But a $-prefix doesn't hurt, so I added it